File: 2029.py
while True:
  try:
    volume = float(input())
    raio = float(input())/2
    altura = volume/(3.14*raio**2)
    print(f"""ALTURA = {altura:.2f}
AREA = {3.14*(raio**2):.2f}""")
  except EOFError:
    break

File: 2029.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    double volume, raio, altura;
    while(cin >> volume){
        cin >> raio;
        raio/=2;
        altura = volume/(3.14*raio*raio);
        
        cout<<fixed<<setprecision(2);
        cout << "ALTURA = " << altura << endl;
        cout << "AREA = " << 3.14*(raio*raio) << endl;
    }
    return 0;
}
